task_2/PoseAndColor[] data